### PR TITLE
chore(asm): update test dependencies for flask and django

### PR DESCRIPTION
**Pull request by christophe-papazian**
appsec tests in django and flask suite depends of files in appsec suite
This PR add the dependencies.

## Checklist

- [x] Change(s) are motivated and described in the PR description.
- [x] Testing strategy is described if automated tests are not included in the PR.
- [x] Risk is outlined (performance impact, potential for breakage, maintainability, etc).
- [x] Change is maintainable (easy to change, telemetry, documentation).
- [x] [Library release note guidelines](https://ddtrace.readthedocs.io/en/stable/releasenotes.html) are followed. If no release note is required, add label `changelog/no-changelog`.
- [x] Documentation is included (in-code, generated user docs, [public corp docs](https://github.com/DataDog/documentation/)).
- [x] Backport labels are set (if [applicable](https://ddtrace.readthedocs.io/en/latest/contributing.html#backporting))

## Reviewer Checklist

- [x] Title is accurate.
- [x] No unnecessary changes are introduced.
- [x] Description motivates each change.
- [x] Avoids breaking [API](https://ddtrace.readthedocs.io/en/stable/versioning.html#interfaces) changes unless absolutely necessary.
- [x] Testing strategy adequately addresses listed risk(s).
- [x] Change is maintainable (easy to change, telemetry, documentation).
- [x] Release note makes sense to a user of the library.
- [x] Reviewer has explicitly acknowledged and discussed the performance implications of this PR as reported in the benchmarks PR comment.
- [x] Backport labels are set in a manner that is consistent with the [release branch maintenance policy](https://ddtrace.readthedocs.io/en/latest/contributing.html#backporting)
- [x] If this PR touches code that signs or publishes builds or packages, or handles credentials of any kind, I've requested a review from `@DataDog/security-design-and-guidance`.
- [x] This PR doesn't touch any of that.

### CODE REVIEW

**Review by emmettbutler — DISMISSED** [review]
For observers, here's an example of these dependencies https://github.com/DataDog/dd-trace-py/blob/10fbd908eefe8e40e657740e42f7104cca85efc0/tests/contrib/django/test_django_appsec.py#L18-L25

**Review by emmettbutler — APPROVED** [review]

File: tests/.suitespec.json [hunk 1]
```diff
@@ -608,6 +608,7 @@
             "@django",
             "@dbapi",
             "@pg",
+            "tests/appsec/*",
```
Comment (P403n1x87) [inline tests/.suitespec.json:611]: 🤔  Why would appsec test sources affect some other component? Isn't the dependency on the appsec component itself enough?

Comment (P403n1x87) [reply tests/.suitespec.json:611]: Ah I think the answer is in https://github.com/DataDog/dd-trace-py/pull/6819#pullrequestreview-1607084399

File: tests/.suitespec.json [hunk 2]
```diff
@@ -648,6 +649,7 @@
             "@flask",
             "@appsec",
             "@wsgi",
+            "tests/appsec/*",
```
Comment (P403n1x87) [inline tests/.suitespec.json:652]: maybe we can move this pattern to the `appsec` component to reduce duplication?

Comment (christophe-papazian) [reply tests/.suitespec.json:652]: For now, we are only supporting django and flask for threats. But this is definitely something we need to keep in mind when we extend asm support.

**Pull request merged**